modules/aws-es/main.tf

```diff
@@ -1,3 +1,4 @@
+#tfsec:ignore:aws-elastic-search-enable-domain-logging tfsec:ignore:aws-elastic-search-enable-logging
```

Comment:
We can ignore this because we are enabling logging, but it's not certain since it's in a dynamic block. Is that right?

Reply:
These look like new rules.. I ran `make lint` in origin/master and got the same problem.
And yes you're right, we may or may not enable logging... Depends on the vars... we are making it customizable.

We could be opinionated here and always have them on too. Or at least the Log_type that would make the tfsec warning go away (AUDIT_LOGS)

@franco-caylent , let us know what you think too.

Reply:
Just to be clear, I tested adding the publishing of AUDIT_LOGS into the tf code and ran `make lint` again and there was no more warnings.

Reply:
I think having the logs being shipped as a default is not really necessary and I don't see any drawbacks from having the ignores in place.

Reply:
Agreed, just want to make sure we have good reasons (i.e. not enforcing logging) to have these ignores.